lib/ansible/modules/cloud/amazon/elb_target_group.py

```diff
@@ -415,6 +415,8 @@ def create_or_update_target_group(connection, module):
     # Get target type
     if module.params.get("target_type") is not None:
         params['TargetType'] = module.params.get("target_type")
```

Comment:
Should not assign TargetType if Boto version doesn't support it and it is "Instance" (the default).